frontend/src/metabase/nav/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -36,7 +36,10 @@ export const AdminNavbar = ({
   const isPaidPlain = useSelector(getIsPaidPlan);
 
   return (
-    <AdminNavbarRoot className="Nav" aria-label={t`Navigation bar`}>
+    <AdminNavbarRoot
+      aria-label={t`Navigation bar`}
+      data-testid="admin-navbar-root"
```

Comment:
I've commented about `data-testid` somewhere. In short there are 2 points I want to be addressed here
1. Don't use `data-testid` for non-test implementation. Using `data` attribute is fine as long as it's not `data-testid` since people could mistake this as only for testing, which is not the truth here.
2. Would be nice to keep the logic the same by not separating between admin navbar and app navbar. My rationale would be to keep things as consistent as possible. This PRs is mainly refactoring so it shouldn't change the implementation and if we could use a single class we could also use a single data attribuse with the same value as well.